<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>613</width>
    <height>766</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Md Juwel Rana</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>39</x>
      <y>20</y>
      <width>541</width>
      <height>681</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="5" column="2">
      <widget class="QPushButton" name="dot">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
font: 75 16pt &quot;MS Shell Dlg 2&quot;;

border-radius:35px;
}

QPushButton:hover{
	
	color: rgb(70, 70, 70);
	background-color: rgb(255, 255, 255);
}

QPushButton{

	color: rgb(255, 255, 255);
	background-color: rgb(70, 70, 70);
}</string>
       </property>
       <property name="text">
        <string>.</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="bog">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
font: 75 16pt &quot;MS Shell Dlg 2&quot;;

border-radius:35px;
}

QPushButton:hover{

	
	color: rgb(185, 61, 0);
	background-color: rgb(255, 255, 255);
}

QPushButton{

	color: rgb(255, 255, 255);
	background-color: rgb(185, 61, 0);
}</string>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="Button_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
font: 75 16pt &quot;MS Shell Dlg 2&quot;;

border-radius:35px;
}

QPushButton:hover{
	
	color: rgb(33, 0, 99);
	background-color: rgb(255, 255, 255);
}

QPushButton{

	color: rgb(255, 255, 255);
	background-color: rgb(33, 0, 99);
}</string>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="Button_1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
font: 75 16pt &quot;MS Shell Dlg 2&quot;;

border-radius:35px;
}

QPushButton:hover{
	
	color: rgb(33, 0, 99);
	background-color: rgb(255, 255, 255);
}

QPushButton{

	color: rgb(255, 255, 255);
	background-color: rgb(33, 0, 99);
}</string>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="gun">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
font: 75 16pt &quot;MS Shell Dlg 2&quot;;

border-radius:35px;
}

QPushButton:hover{

	
	color: rgb(185, 61, 0);
	background-color: rgb(255, 255, 255);
}

QPushButton{

	color: rgb(255, 255, 255);
	background-color: rgb(185, 61, 0);
}</string>
       </property>
       <property name="text">
        <string>*</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="Button_5">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
font: 75 16pt &quot;MS Shell Dlg 2&quot;;

border-radius:35px;
}

QPushButton:hover{
	
	color: rgb(33, 0, 99);
	background-color: rgb(255, 255, 255);
}

QPushButton{

	color: rgb(255, 255, 255);
	background-color: rgb(33, 0, 99);
}</string>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="Button_6">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
font: 75 16pt &quot;MS Shell Dlg 2&quot;;

border-radius:35px;
}

QPushButton:hover{
	
	color: rgb(33, 0, 99);
	background-color: rgb(255, 255, 255);
}

QPushButton{

	color: rgb(255, 255, 255);
	background-color: rgb(33, 0, 99);
}</string>
       </property>
       <property name="text">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QPushButton" name="soman">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
font: 75 16pt &quot;MS Shell Dlg 2&quot;;

border-radius:35px;
}

QPushButton:hover{
	color: rgb(0, 58, 0);
	background-color: rgb(255, 255, 255);
}

QPushButton{

	color: rgb(255, 255, 255);
	background-color: rgb(0, 58, 0);
}</string>
       </property>
       <property name="text">
        <string>=</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="Button_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
font: 75 16pt &quot;MS Shell Dlg 2&quot;;

border-radius:35px;
}

QPushButton:hover{
	
	color: rgb(33, 0, 99);
	background-color: rgb(255, 255, 255);
}

QPushButton{

	color: rgb(255, 255, 255);
	background-color: rgb(33, 0, 99);
}</string>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="zog">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
font: 75 16pt &quot;MS Shell Dlg 2&quot;;

border-radius:35px;
}

QPushButton:hover{

	
	color: rgb(185, 61, 0);
	background-color: rgb(255, 255, 255);
}

QPushButton{

	color: rgb(255, 255, 255);
	background-color: rgb(185, 61, 0);
}</string>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="Button_8">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
font: 75 16pt &quot;MS Shell Dlg 2&quot;;

border-radius:35px;
}

QPushButton:hover{
	
	color: rgb(33, 0, 99);
	background-color: rgb(255, 255, 255);
}

QPushButton{

	color: rgb(255, 255, 255);
	background-color: rgb(33, 0, 99);
}</string>
       </property>
       <property name="text">
        <string>8</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="Del">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
font: 75 16pt &quot;MS Shell Dlg 2&quot;;

border-radius:35px;
}

QPushButton:hover{

	
	
	color: rgb(88, 0, 0);
	background-color: rgb(255, 255, 255);
}

QPushButton{

	color: rgb(255, 255, 255);
	background-color: rgb(88, 0, 0);
}</string>
       </property>
       <property name="text">
        <string>Del</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="Button_7">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
font: 75 16pt &quot;MS Shell Dlg 2&quot;;

border-radius:35px;
}

QPushButton:hover{
	
	color: rgb(33, 0, 99);
	background-color: rgb(255, 255, 255);
}

QPushButton{

	color: rgb(255, 255, 255);
	background-color: rgb(33, 0, 99);
}</string>
       </property>
       <property name="text">
        <string>7</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QPushButton" name="vag">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
font: 75 16pt &quot;MS Shell Dlg 2&quot;;

border-radius:35px;
}

QPushButton:hover{

	
	color: rgb(185, 61, 0);
	background-color: rgb(255, 255, 255);
}

QPushButton{

	color: rgb(255, 255, 255);
	background-color: rgb(185, 61, 0);
}</string>
       </property>
       <property name="text">
        <string>/</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="Button_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
font: 75 16pt &quot;MS Shell Dlg 2&quot;;

border-radius:35px;
}

QPushButton:hover{
	
	color: rgb(33, 0, 99);
	background-color: rgb(255, 255, 255);
}

QPushButton{

	color: rgb(255, 255, 255);
	background-color: rgb(33, 0, 99);
}</string>
       </property>
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="Button_9">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
font: 75 16pt &quot;MS Shell Dlg 2&quot;;

border-radius:35px;
}

QPushButton:hover{
	
	color: rgb(33, 0, 99);
	background-color: rgb(255, 255, 255);
}

QPushButton{

	color: rgb(255, 255, 255);
	background-color: rgb(33, 0, 99);
}</string>
       </property>
       <property name="text">
        <string>9</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QPushButton" name="Button_0">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
font: 75 16pt &quot;MS Shell Dlg 2&quot;;

border-radius:35px;
}

QPushButton:hover{
	
	color: rgb(33, 0, 99);
	background-color: rgb(255, 255, 255);
}

QPushButton{

	color: rgb(255, 255, 255);
	background-color: rgb(33, 0, 99);
}</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QPushButton" name="Clear">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
font: 75 16pt &quot;MS Shell Dlg 2&quot;;

border-radius:35px;
}

QPushButton:hover{

	
	color: rgb(255, 0, 0);
	background-color: rgb(255, 255, 255);
}

QPushButton{

	color: rgb(255, 255, 255);
	background-color: rgb(255, 0, 0);
}</string>
       </property>
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="4">
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">font: 75 32pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="lineWidth">
        <number>3</number>
       </property>
       <property name="midLineWidth">
        <number>3</number>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>613</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
